bassist/bassist/parser/log_file/debs_stdout.py

```diff
@@ -2,10 +2,11 @@
 # See the file LICENSE for copying permission.
 from . import common
 from ...systems import package
-from results import store_debs
-from run_sql import execute_sql
+
+debs = []
```

Comment:
Sorry I got it wrong. Here's the updated image:
<img width="696" alt="screen shot 2017-08-12 at 11 47 14 am" src="https://user-images.githubusercontent.com/134408/29242140-2be2d45a-7f55-11e7-82a0-f7c9a3be9c62.png">